logstash-core/src/main/java/org/logstash/log/LoggerExt.java

```diff
@@ -0,0 +1,196 @@
+    }
+
+    @JRubyMethod(name = "debug", required = 1, optional = 1)
+    public IRubyObject rubyDebug(final ThreadContext context, final IRubyObject[] args) {
+        if (args.length > 1) {
+            logger.debug(args[0].asJavaString(), args[1]);
+        } else {
+            logger.debug(args[0].asJavaString());
+        }
+        return this;
+    }
+
+    @JRubyMethod(name = "warn", required = 1, optional = 1)
+    public IRubyObject rubyWarn(final ThreadContext context, final IRubyObject[] args) {
+        if (args.length > 1) {
+            logger.warn(args[0].asJavaString(), args[1]);
+        } else {
+            logger.warn(args[0].asJavaString());
+        }
+        return this;
+    }
+
+    @JRubyMethod(name = "info", required = 1, optional = 1)
+    public IRubyObject rubyInfo(final ThreadContext context, final IRubyObject[] args) {
+        if (args.length > 1) {
+            logger.info(args[0].asJavaString(), args[1]);
+        } else {
+            logger.info(args[0].asJavaString());
+        }
+        return this;
+    }
+
+    @JRubyMethod(name = "error", required = 1, optional = 1)
+    public IRubyObject rubyError(final ThreadContext context, final IRubyObject[] args) {
+        if (args.length > 1) {
+            logger.error(args[0].asJavaString(), args[1]);
+        } else {
+            logger.error(args[0].asJavaString());
+        }
+        return this;
+    }
+
+    @JRubyMethod(name = "fatal", required = 1, optional = 1)
+    public IRubyObject rubyFatal(final ThreadContext context, final IRubyObject[] args) {
+        if (args.length > 1) {
+            logger.fatal(args[0].asJavaString(), args[1]);
+        } else {
+            logger.fatal(args[0].asJavaString());
+        }
+        return this;
+    }
+
+    @JRubyMethod(name = "trace", required = 1, optional = 1)
+    public IRubyObject rubyTrace(final ThreadContext context, final IRubyObject[] args) {
+        if (args.length > 1) {
+            logger.trace(args[0].asJavaString(), args[1]);
+        } else {
+            logger.trace(args[0].asJavaString());
+        }
+        return this;
+    }
+
+    @JRubyMethod(name = "configure_logging", meta = true, required = 1, optional = 1)
+    public static IRubyObject configureLogging(final ThreadContext context, final IRubyObject self,
+                                        final IRubyObject args[]) {
+        synchronized (CONFIG_LOCK) {
+            RubyString path = args.length > 1 ? (RubyString) args[1] : null;
+            String level = args[0].asJavaString();
+            try {
+                setLevel(level, (path == null || path.isNil()) ? null : path.asJavaString());
+            } catch (Exception e) {
+                throw new IllegalArgumentException(
+                        String.format("invalid level[%s] for logger[%s]", level, path));
+            }
+
+        }
+        return context.nil;
+    }
+
+    @JRubyMethod(name = "reconfigure", meta = true)
```

Comment:
@danhermann NIT: you don't need to give the `"name"` if it's the same as the Java method name :)